pom.xml

```diff
@@ -27,6 +27,7 @@
     <module>parboiled</module>
     <module>pegdown</module>
     <module>saxon</module>
+    <module>saxon-98</module>
```

Comment:
Do we need to keep both `saxon` and `saxon-98` modules?

Reply:
Created reminder: https://github.com/daisy/osgi-libs/issues/46